From: uazo <uazo@users.noreply.github.com>
Date: Fri, 2 May 2025 14:20:38 +0000
Subject: Disable CSSDynamicRangeLimit

Disables the ability for pages to limit the maximum brightness of HDR content
since use as screen fingerprinting is possible.

see https://www.github.com/w3ctag/design-reviews/issues/1027

License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
---
 .../blink/renderer/platform/runtime_enabled_features.json5      | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
@@ -1250,7 +1250,7 @@
     },
     {
       name: "CSSDynamicRangeLimit",
-      status: "stable",
+      status: "experimental",
     },
     {
       // Include custom properties in CSSComputedStyleDeclaration::item/length.
--
